### PR TITLE
address Rafael's comments

### PR DESCRIPTION
**Pull request by kainino0x**

### CODE REVIEW

File: sdk/tests/conformance/misc/webgl-specific-stencil-settings.html [hunk 1]
```diff
@@ -44,13 +44,6 @@
 description("Tests that stencil mask/func are validated correctly when the front state and back state differ.");
 
 var gl = wtu.create3DContext();
-var program = wtu.loadStandardProgram(gl);
-gl.useProgram(program);
-var vertexObject = gl.createBuffer();
-gl.bindBuffer(gl.ARRAY_BUFFER, vertexObject);
-gl.enableVertexAttribArray(0);
-gl.vertexAttribPointer(0, 4, gl.FLOAT, false, 0, 0);
-wtu.glErrorShouldBe(gl, gl.NO_ERROR, "Setup should succeed");
```
Comment (kdashg) [inline sdk/tests/conformance/misc/webgl-specific-stencil-settings.html:53]: What are you replacing this with? I don't see where we create the program now.

Comment (kainino0x) [reply sdk/tests/conformance/misc/webgl-specific-stencil-settings.html:53]: Hm, I guess we do strictly speaking need a program even though the draws are noops.
This must be another early-out edge case, because it's passing in Chrome.
I'll just add the normal setupUnitQuad etc calls. [EDIT: actually probably just the program]

Comment (kdashg) [reply sdk/tests/conformance/misc/webgl-specific-stencil-settings.html:53]: It will need a program, and every active attribute with enabled-array will need to type-match with the program. (We can probably just use generics?) It's probably worth adding a wtu.dummyDrawCall() that lazily creates a no-attrib program and calls drawArrays(TRIANGLES, 0, 0).

Comment (kainino0x) [reply sdk/tests/conformance/misc/webgl-specific-stencil-settings.html:53]: Done

Comment (kainino0x) [reply sdk/tests/conformance/misc/webgl-specific-stencil-settings.html:53]: Oops, missed last comment. Let me see about doing that.

Comment (kainino0x) [reply sdk/tests/conformance/misc/webgl-specific-stencil-settings.html:53]: > it's passing in Chrome

It actually wasn't passing in Chrome. I must have loaded a cached version.

Comment (kainino0x) [reply sdk/tests/conformance/misc/webgl-specific-stencil-settings.html:53]: Okay, that's done now too.

**Review by kdashg — APPROVED** [review]
